start.py

```diff
@@ -1029,6 +1037,9 @@ def boot():
             message = str(exc)
             _print(message)
             if os.environ.get("OPENPYPE_HEADLESS_MODE") == "1":
+                openpype_versions = bootstrap.find_openpype(
+                    include_zips=True, staging=use_staging
+                )
                 list_versions(openpype_versions, local_version)
             else:
                 igniter.show_message_dialog("Version not found", message)
```

Comment:
I'm thinking that since this logic is duplicated just below, should we maybe encapsulate that in its own function, like:

```python
def _debug_version_not_found(exc):
    message = str(exc)
    _print(message)
    if os.environ.get("OPENPYPE_HEADLESS_MODE") == "1":
        openpype_versions = bootstrap.find_openpype(include_zips=True, 
                                                    staging=use_staging)
        list_versions(openpype_versions, local_version)
    else:
        igniter.show_message_dialog("Version not found", message)
```

We could even define this within the boot() function just before this part in the code - just so it's clear that it's re-used?